GrandCentralBoard/App/ViewControllers/MainViewController.swift

```diff
@@ -7,34 +7,43 @@ import UIKit
 import GrandCentralBoardCore
 
 
-class MainViewController: UIViewController {
+final class MainViewController: UIViewController {
 
-    private var board: GrandCentralBoard?
+    private let autoStack = AutoStack()
+    private let scheduler = Scheduler()
+    private let dataDownloader = DataDownloader()
+    private var configurationRefresher: ConfigurationRefresher?
 
-    var configuration: Configuration! {
-        didSet {
-            setUpBoardWithConfiguration(configuration)
-        }
-    }
+    private lazy var board: GrandCentralBoard = { GrandCentralBoard(scheduler: self.scheduler, stack: self.autoStack) }()
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        setUpBoardWithConfiguration(configuration)
-    }
+    private lazy var availableBuilders: [WidgetBuilding] = [
+        WatchWidgetBuilder(dataDownloader: self.dataDownloader),
+        BonusWidgetBuilder(dataDownloader: self.dataDownloader),
+        GoogleCalendarWatchWidgetBuilder(),
+        HarvestWidgetBuilder(),
+        ImageWidgetBuilder(dataDownloader: self.dataDownloader),
+        BlogPostsPopularityWidgetBuilder()
+    ]
 
-    private func setUpBoardWithConfiguration(configuration: Configuration) {
+    private lazy var configurationFetching: ConfigurationFetching = {
```

Comment:
There's a lot of lazy variables here, which doesn't improve anything, but adds complexity to following initialisation order . Can they be initialised explicitly?